clusterman/draining/queue.py

```diff
@@ -415,34 +434,44 @@ def host_from_instance_id(
         logger.warning(f"No instance data found for {instance_id}")
         return None
     try:
-        sfr_ids = [tag["Value"] for tag in instance_data[0]["Tags"] if tag["Key"] == "aws:ec2spot:fleet-request-id"]
+        group_ids = [
+            tag["Value"]
+            for tag in instance_data[0]["Tags"]
+            if tag["Key"] in {"aws:ec2spot:fleet-request-id", "aws:ec2:fleet-id", }
+        ]
         scheduler = "mesos"
         for tag in instance_data[0]["Tags"]:
             if tag["Key"] == "KubernetesCluster":
                 scheduler = "kubernetes"
                 break
     except KeyError as e:
-        logger.warning(f"SFR tag key not found: {e}")
-        sfr_ids = []
-    if not sfr_ids:
-        logger.warning(f"No SFR ID found for {instance_id}")
+        logger.warning(f"SFR/SF tag key not found: {e}")
+        group_ids = []
+    if not group_ids:
+        logger.warning(f"No SFR/SF ID found for {instance_id}")
```

Comment:
thoughts on rewording this a tiny bit:
```suggestion
        logger.exception(f"Spot tag key not found - is this Spot Fleet/ASG correctly configured?")
        group_ids = []
    if not group_ids:
        logger.warning(f"Not draining {instance_id}: no Spot ID found - is this actually a Spot instance?")
```

might clear things up for new folks looking at these logs